companion/utils/tasks.py

```diff
@@ -70,9 +72,13 @@ def update_source_database(variable, source):
     ##### ! TODO UPDATE .get_graphql_data (located in utils/datasources.py)
     ##### ! This is a class specific function that parses the raw tool data into a graphql standard
     graphql_data = source.get_graphql_data()

```

Comment:
You're currently only interested in temperature data right? If that's the case you might want to do something like:
```
if is_instance(source, location_data):
    .... then you code ...
```